flusio/Minz
# imp: Allow to inherit Configuration to declare application types

## How to test manually



1. Create a Minz app and setup PHPStan
2. Declare a `\App\Configuration` class as described in the doc
3. In the app, access an application option
4. Verify that PHPStan doesn't complain about the type of the option

## Reviewer checklist
